dydx/constants.py

```diff
@@ -57,12 +58,15 @@
 SMALL_TRADE_SIZE_DAI = 10000 * (10 ** DECIMALS_DAI)
 SMALL_TRADE_SIZE_PBTC = 0.5 * (10 ** DECIMALS_PBTC)
 FEE_SMALL_WETH = Decimal(100) * FROM_BIPS
-FEE_LARGE_WETH = Decimal(15) * FROM_BIPS
+FEE_LARGE_WETH = Decimal(100) * FROM_BIPS
 FEE_SMALL_DAI = Decimal(100) * FROM_BIPS
-FEE_LARGE_DAI = Decimal(5) * FROM_BIPS
-FEE_LARGE_PBTC = Decimal(7.5) * FROM_BIPS
-FEE_SMALL_PBTC = Decimal(50) * FROM_BIPS
-FEE_MAKER_PBTC = Decimal(-2.5) * FROM_BIPS
+FEE_LARGE_DAI = Decimal(100) * FROM_BIPS
+FEE_LARGE_PBTC = Decimal(100) * FROM_BIPS
+FEE_SMALL_PBTC = Decimal(100) * FROM_BIPS
+FEE_MAKER_PBTC = Decimal(100) * FROM_BIPS
+FEE_LARGE_WETH_PUSD = Decimal(100) * FROM_BIPS
+FEE_SMALL_WETH_PUSD = Decimal(100) * FROM_BIPS
+FEE_MAKER_WETH_PUSD = Decimal(100) * FROM_BIPS
```

Comment:
These new variables are never used, instead you can modify the `get_limit_fee()` function